.github/workflows/page-age.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Page age
+
+# Controls when the workflow will run
+on:
+  schedule:
+    # Run this once per day, towards the end of the day for keeping the most
+    # recent data point most meaningful (hours are interpreted in UTC).
+    - cron: "55 23 * * 0"
```

Comment:
Page age only gets non-trivial updates, when there are changes to the repo, should we perhaps change the trigger to:
```
on:
  push:
    - maser
```